doc/man3/ERR_get_error.pod

```diff
@@ -65,42 +63,48 @@ See L<ERR_GET_LIB(3)> for obtaining further specific information
+An unset function name is indicated as "", i.e. an empty string.
 
 A pointer returned this way by these functions and the ones below
-is valid until the respective entry is removed from the error queue.
-
-ERR_get_error_func(), ERR_peek_error_func() and
-ERR_peek_last_error_func() are the same as ERR_get_error(),
-ERR_peek_error() and ERR_peek_last_error(), but on success they
-additionally store the name of the function where the error occurred
-in *B<func>, unless it is B<NULL>.
-An unset function name is indicated as B<"">.
-
-ERR_get_error_data(), ERR_peek_error_data() and
-ERR_peek_last_error_data() are the same as ERR_get_error(),
-ERR_peek_error() and ERR_peek_last_error(), but on success they
-additionally store additional data and flags associated with the error
-code in *B<data> and *B<flags>, as far as they are not B<NULL>.
-Unset data is indicated as B<"">.
-In this case the value given for the flag is irrelevant (and equals B<0>).
-*B<data> contains a string if *B<flags>&B<ERR_TXT_STRING> is true.
-
-ERR_get_error_all(), ERR_peek_error_all() and
-ERR_peek_last_error_all() are combinations of all of the above.
-
-ERR_get_error_line_data(), ERR_peek_error_line_data() and
-ERR_peek_last_error_line_data() are older variants of ERR_get_error_all(),
-ERR_peek_error_all() and ERR_peek_last_error_all(), and should no longer
-be used.
-
-An application B<MUST NOT> free the *B<data> pointer (or any other pointers
+is valid until the respective entry is overwritten in the error queue.
+
+ERR_peek_error_line() and ERR_peek_last_error_line() are the same as
+ERR_peek_error() and ERR_peek_last_error(), but on success they additionally
+store the filename and line number where the error occurred in *I<file> and
+*I<line>, as far as they are not NULL. 
+An unset filename is indicated as "", i.e., an empty string.
+An unset line number is indicated as 0.
+
+ERR_peek_error_func() and ERR_peek_last_error_func() are the same as
+ERR_peek_error() and ERR_peek_last_error(), but on success they additionally
+store the name of the function where the error occurred in *I<func>, unless
+it is NULL. 
+An unset function name is indicated as "".
+
+ERR_peek_error_data() and ERR_peek_last_error_data() are the same as
+ERR_peek_error() and ERR_peek_last_error(), but on success they additionally
+store additional data and flags associated with the error code in *I<data>
+and *I<flags>, as far as they are not NULL.
+Unset data is indicated as "".
+In this case the value given for the flag is irrelevant (and equals 0).
+*I<data> contains a string if *I<flags>&B<ERR_TXT_STRING> is true.
+
+ERR_peek_error_all() and ERR_peek_last_error_all() are combinations of all
+of the above.
+
+ERR_get_error_line(), ERR_get_error_line_data(), ERR_peek_error_line_data()
+and ERR_peek_last_error_line_data() are older variants of ERR_get_error_all(),
+ERR_peek_error_all() and ERR_peek_last_error_all(), and may give confusing
+results.  They should no longer be used.
```

Comment:
This should mention that they are deprecated.

Reply:
Done